spack/daint/compilers.yaml

```diff
@@ -63,6 +63,8 @@ compilers:
     modules:
     - PrgEnv-intel
     - intel/19.0.1.144
+    - cdt/20.08
+    - intel/19.0.1.144
```

Comment:
You have added this twice

Reply:
Yes! Due to Spack's limitation, you cannot have the `cdt` defined before the first `intel` entry. And whenever we load the `cdt` module the compiler is reset. So we need to load again the `intel` module.